Doc/examples/nmr/simplepredict.py

```diff
@@ -123,14 +122,14 @@
     if noe1 != "":
 
         # Here I'm using the XpkEntry class to gain access to
-        # specific fields in the that make the information
+        # specific fields in the file that make the information
         # more readable and suitable for creating data tables
         # This output will be printed to the screen.
         # The data table contains the assignment, coordinates and
         # intensity of the resonance.
 
-        print(string.split(entry1.fields["15N2.L"], ".")[0], "-->",
-              string.split(entry1.fields["N15.L"], ".")[0], "\t",
+        print(str.split(entry1.fields["15N2.L"], ".")[0], "-->",
```

Comment:
You are right the old code using the ``string`` module ought to be updated, but why not use the string method?:

```
    print(entry1.fields["15N2.L"].split("."), "-->",
```

Reply:
It causes an AttributeError: module 'string' has no attribute 'split'; so had to change it.

Reply:
Sorry, I had a typo amending now.

Reply:
Heh. I had a typo too, missed out the ``[0]`` bit.